styles.css

```diff
@@ -219,17 +225,16 @@ section {
   }
 }
 
-/* -------------------- sec1 ------------------------- */
+/* -------------------- main-information-section ------------------------- */
 
-.sec1 {
+.main-information-section {
   position: relative;
   display: flex;
-  flex-direction: row;
+  flex: row;
   justify-content: flex-start;
   align-items: flex-start;
   box-sizing: border-box;
-  padding: 0px var(--padding-x);
-  padding-top: 80px;
+  padding: 80px var(--padding-x) 0px;
```

Comment:
This is wrong.

Me commenting like this is wrong.  😁

I can tell you in another way but just to let you know how the bad review might be.

THIS IS NOT WRONG 🙄.